tests/unit/services/sharing-service-test.js

```diff
@@ -20,120 +20,6 @@ test('it exists', function (assert) {
   assert.ok(service);
 });
 
-// Replace this with your real tests.
```

Comment:
equivalent tests can be found in [`arcgis-rest-sharing/test/access.test.ts`](https://github.com/Esri/arcgis-rest-js/blob/b0d76f9a4e7396e4f05bb4948078ae96b724d9b8/packages/arcgis-rest-sharing/test/access.test.ts)